helm/prometheus-meta-operator/ci/default-values.yaml

```diff
@@ -4,6 +4,9 @@ Installation:
     Monitoring:
       Prometheus:
         Host: ""
+        Storage:
+          Enabled: true
+          CreatePVC: true
```

Comment:
What's the use-case for two separate settings?

AFAICS both have to be true for it to create the PVC, right? If so could we simplify this to just `CreatePVC` and likewise rename the `isPersistent` and `SupportsPersistentStorage` to `CreatePVC`?

I'd rather the CLI flag and struct attributes reflect what it's actually going to do, rather than use an indirection.

Reply:
This is how it is configured under installations so I am not sure we can change it however we want but I can rename it to create PVC instead